build.gradle.kts

```diff
@@ -17,8 +17,11 @@ repositories {
 dependencies {
     implementation("com.google.protobuf:protobuf-java:2.6.1")
     implementation("org.jetbrains.kotlin:kotlin-stdlib")
+
+    val kotlinx_metadata_version: String by project
+    implementation("org.jetbrains.kotlinx:kotlinx-metadata-jvm:$kotlinx_metadata_version")
```

Comment:
Not related to strict api mode, just noticed that metadata-jvm version was specified in two places (in here and an unused property in `gradle.properties`)